src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-overview-dashboard/rgw-overview-dashboard.component.ts

```diff
@@ -91,6 +113,32 @@ export class RgwOverviewDashboardComponent implements OnInit, OnDestroy {
       this.rgwZoneCount = data['zones'].length;
     });
     this.getPrometheusData(this.prometheusService.lastHourDateObject);
+    this.multisiteSyncStatus$ = this.subject.pipe(
+      switchMap(() =>
+        this.rgwMultisiteService.getSyncStatus().pipe(
+          tap((data: any) => {
+            this.loading = false;
+            this.replicaZonesInfo = data['dataSyncInfo'];
+            this.metadataSyncInfo = data['metadataSyncInfo'];
+            [this.realm, this.zonegroup, this.zone] = data['primaryZoneData'];
+            for (const obj of this.replicaZonesInfo) {
+              if (obj['syncstatus'] === 'Not Syncing From Zone') {
+                obj['syncstatus'] = 'Not Syncing';
+              }
+            }
+          })
+        )
+      ),
+      tap(() => {
+        const zonegroup = new RgwZonegroup();
+        zonegroup.name = this.zonegroup;
+        this.rgwZonegroupService.get(zonegroup).subscribe((data: any) => {
+          this.showMultisiteCard = data['zones'].length !== 1;
+          this.syncCardLoading = false;
+        });
+      }),
+      shareReplay(1)
+    );
```

Comment:
if the switchMap is not working, then mergeMap would be it! and the `data` inside the tap is the return of zonegroupService, because we are returning it in the first mergeMap.

That reminds could you also rename the variables to more meaninful ones!
```suggestion
     this.multisiteSyncStatus$ = this.subject.pipe(
      mergeMap(() =>
        this.rgwMultisiteService.getSyncStatus().pipe(
          mergeMap((data: any) => {
            this.loading = false;
            this.replicaZonesInfo = data['dataSyncInfo'];
            this.metadataSyncInfo = data['metadataSyncInfo'];
            [this.realm, this.zonegroup, this.zone] = data['primaryZoneData'];
            const zonegroup = new RgwZonegroup();
            zonegroup.name = this.zonegroup;
            return this.rgwZonegroupService.get(zonegroup)
          })
        ),
        tap((data: any) => {
          this.showMultisiteCard = data['zones'].length !== 1;
          this.syncCardLoading = false;
        })
      ),
      shareReplay(1)
    );
```

Reply:
This didn't work as well